src/test/java/org/example/automation/amazon/VerifyFilterSearchTest.java

```diff
@@ -0,0 +1,137 @@
+package org.example.automation.amazon;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindAll;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class VerifyFilterSearchTest extends BaseWebDriverTest{
+    public static final String WEB_ADDRESS="https://www.amazon.com";
```

Comment:
Constant "WEB_ADDRESS" is not used.

Reply:
Used the constant on the latest commit. Thank you